### PR TITLE
Fix typo at most_allocated_test.go

### PR DESCRIPTION
**Pull request by yoongon**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
For more clarification, let's fix typo.
Memory on node is 9_000(not 8_000), and requested memory at pod is 9_000(not 8_000).

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func TestMostAllocatedScoringStrategy(t *testing.T) {
 			// Node1 Score: (100 + 90) / 2 = 95
 			// Node2 scores on 0-MaxNodeScore scale
 			// CPU Score: (5000 * MaxNodeScore) / 10000 = 50
-			// Memory Score: 8000 *MaxNodeScore / 8000 return 100
+			// Memory Score: 9000 *MaxNodeScore / 9000 return 100
```
Comment (denkensk) [inline pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go:157]:
```suggestion
			// Memory Score: 9000 * MaxNodeScore / 9000 return 100
```

**Review by kerthcet — COMMENTED** [review]
/lgtm

**Comment by kerthcet** [comment]
Great catch, thanks @yoongon !

File: pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go (hunk 1 above)
Comment (kerthcet) [reply pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go:157]: Plz follow @denkensk advices. 🥲

Comment (yoongon) [reply pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go]: Thanks for checking.
Done!

**Comment by yoongon** [comment]
/assign alculquicondor
cc @alculquicondor

**Comment by yoongon** [comment]
@kerthcet 
Thanks for your checking!
I've added one more commit to follow the advice.
Please check again :)

**Comment by denkensk** [comment]
> @kerthcet Thanks for your checking! I've added one more commit to follow the advice. Please check again :)

Pls squash to one. Thanks.

**Comment by yoongon** [comment]
> Pls squash to one. Thanks.
@denkensk 
I make it one, but it seems pending now.
Thanks.

**Pull request merged**